selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginWithForkTest.java

```diff
@@ -60,6 +60,12 @@
   @Inject private Ide ide;
   @Inject private TestUser productUser;
 
+  //  @Inject private TestGitHubRepository testRepo;
+
```

Comment:
Why commenting is used?

Reply:
This is an example of used in target test.

Reply:
Should you uncomment this rows? And in this case test maintainer should only check and approve your changes and doesn`t do anything more.

Reply:
If I uncomment it for now, it will be creating two unnecessary repositories during the execution while the test will be fixed.

Reply:
I would remove that code and add en example in the PR description.